tests/nlu/classifiers/test_mitie_intent_classifier.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+
+from rasa.engine.graph import ExecutionContext
+from rasa.engine.storage.resource import Resource
+from rasa.engine.storage.storage import ModelStorage
+from rasa.nlu.classifiers.mitie_intent_classifier import (
+    MitieIntentClassifierGraphComponent,
+)
+from rasa.nlu.tokenizers.mitie_tokenizer import MitieTokenizer
+
+from rasa.nlu.utils.mitie_utils import MitieModel, MitieNLPGraphComponent
+import rasa.shared.nlu.training_data.loading
+from rasa.shared.nlu.constants import (
+    TEXT,
+    INTENT,
+    INTENT_NAME_KEY,
+    PREDICTED_CONFIDENCE_KEY,
+)
+from rasa.shared.nlu.training_data.message import Message
+
+
+@pytest.fixture
+def mitie_model(
+    default_model_storage: ModelStorage, default_execution_context: ExecutionContext
+) -> MitieModel:
+    component = MitieNLPGraphComponent.create(
+        MitieNLPGraphComponent.get_default_config(),
+        default_model_storage,
+        Resource("mitie"),
+        default_execution_context,
+    )
+
+    return component.provide()
+
+
+def test_train_load_predict_loop(
+    default_model_storage: ModelStorage,
+    default_execution_context: ExecutionContext,
+    mitie_model: MitieModel,
+):
+    resource = Resource("mitie_classifier")
+    component = MitieIntentClassifierGraphComponent.create(
+        MitieIntentClassifierGraphComponent.get_default_config(),
+        default_model_storage,
+        resource,
+        default_execution_context,
+    )
+
+    training_data = rasa.shared.nlu.training_data.loading.load_data(
+        "data/examples/rasa/demo-rasa.yml"
+    )
+    tokenizer = MitieTokenizer()
+    # Tokenize message as classifier needs that
+    tokenizer.train(training_data)
```

Comment:
as far as I know that's already on our radar. `MitieTokenizer` just to be adapted so that the method is no longer called `train` but rather `process_train` or something like that.

Reply:
perfect :) looking forward to this since I've first seen the featurizers :D